backend/apps/billing/views.py

```
@@ -34,3 +34,56 @@ def get(self, request):
+            bounty=bounty,
+            user=request.user
+        )
+        if not created:
+            return Response({"status": "already_claimed", "claim_id": claim.id})
+            
+        return Response({"status": "claimed", "claim_id": claim.id})
+
+class BountyClaimViewSet(viewsets.ModelViewSet):
+    queryset = BountyClaim.objects.all()
+    serializer_class = BountyClaimSerializer
+    permission_classes = [permissions.IsAuthenticated]
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)